workflow/scripts/rule_zarr_stats.py

```
@@ -0,0 +1,274 @@
+    fancybox=True,
+    shadow=True,
+    ncol=4,
+)
+# fig.tight_layout()
+fig.savefig(
+    f"{snakemake.config['data_dir']}/zarr_stats/"
+    f"{snakemake.wildcards.subset_name}-{snakemake.wildcards.region_name}-{snakemake.wildcards.filter_set}/"
+    f"filter-fractions.png"
+)
+
+filter_config = snakemake.config["filters"][snakemake.wildcards.filter_set]
```

Ruff (0.14.5)

Undefined name `snakemake`

(F821)